content/applications/inventory_and_mrp/maintenance/maintenance_reporting.rst

```diff
@@ -0,0 +1,295 @@
+=====================
+Maintenance reporting
+=====================
+
+Odoo *Maintenance* tracks both current and past maintenance requests in a reporting dashboard. Users
+can access the *Maintenance Requests* report to measure information in different reporting views to
+help make informed decisions about equipment, and plan corrective and preventive maintenance.
+
+.. _maintenance/reporting/dashboard:
+
+Reporting dashboard
+===================
+
+To view the *Maintenance* reporting dashboard, navigate to :menuselection:`Maintenance app -->
+Reporting --> Maintenance Requests`. Doing so displays the :guilabel:`Maintenance Requests` report
+overview.
+
+By default, the report is displayed in the :guilabel:`Graph` view, in :guilabel:`Bar Chart` form,
+with the :guilabel:`Stacked` data display.
+
+To read about view types, skip to the :ref:`Reporting views <maintenance/reporting/view-types>`
+section.
+
+From the :guilabel:`Maintenance Requests` report overview, click :guilabel:`Measures`, and select
+how to measure the information. By default, the :guilabel:`Count` measure is displayed.
+
+Next to :guilabel:`Measures`, click :guilabel:`Insert in Spreadsheet` to open a pop-up window
+reading :guilabel:`Select a spreadsheet to insert your graph.`. Select a spreadsheet from the
+:guilabel:`Spreadsheets` tab, or click the :guilabel:`Dashboards` tab, and select a dashboard to
+insert the graph. Once ready, click :guilabel:`Confirm` to close the pop-up window.
+
+.. image:: maintenance_reporting/maintenance-reporting-spreadsheet-popup.png
+   :align: center
+   :alt: Select a spreadsheet to insert your graph pop-up window.
+
+Filters, Group By, and Favorites
```

Comment:
You can add a link to the basic documentation at the end of this section about the reporting views, etc.

Reply:
Nice idea - added. Suggestion implemented ✅